### DISCUSSION TITLE
Newbie: Artisan and Hottop 2K+ Not Communicating

### DISCUSSION THREAD

**Discussion by jwarthman** [artisan-roaster-scope/artisan#771] [Q&A] [answered]
Macbook Pro running macOS Catalina 10.15.7 and Artisan 2.4.6 build 2 (f1b8e57). 

I just received my new Hottop KN-8828B-2K+ today, and of course immediately tried to connect it with my Mac - no joy. 

Here's what I've done:
1. connect the USB-A cable between the Hottop and the Mac.
2. Launch Artisan.
3. Reset Artisan to get to a known starting point.
4. Config > Machine > Artisan to select my machine type.
- When prompted for port select /dev/cu.usbserial-DA01POGR
- The only other choice is FT230X Basic UART
5. In the upper left corner of the Artisan window is the message "Artisan configured for Hottop KN-8828B-2K+"
6. On the Hottop, press the ON/OFF START button (the Hottop was "Off" for the above steps.) 

From here I've tried various combinations of Hottop Auto and Manual modes, with and without pressing the Artisan "On" button.

At no time in this process have I seen the Hottop's "USB" icon on its display.

I suspect there's something really basic that I'm doing (or not), and hope someone can spot the issue or suggest things I might try.

TIA,
Jim

**Comment by MAKOMO** [comment]
Did you try the other option: "FT230X Basic UART"?

**Comment by bigl00z3** [comment, marked as answer]
Hello Jim,

I imagine you have followed the steps on [https://artisan-roasterscope.blogspot.com/2015/05/hottop-kn-8828b-2k.html](url)

First you should check if the issue is with your configuration in Artisan or already in the connection in between the Roaster and the Mac (Artisan cannot communicate if the Roaster is not connected correctly to the computer). For this:

I f you go in the Apple menu:
_/About This Mac/ System Report_

In the "_Hardware_" selection, click on "_USB_". 

Do you see the Roaster being connected.
When you plug/unplug the usb cable you will have to do a _File/Refresh Information_ to update the list (or ⌘ R ).

If you don't see it here, the problem is not with Artisan but the connection in between the Roaster and the computer. 
Eventually change the USB cable. 

Once you know that the computer sees the roaster. Try to configure it again.

On my side, I don't have a hot top, but when I select the Config/Machine/Hottop configuration I get the same device proposed (which makes me think that you are not selecting the Hottop):
![image](https://user-images.githubusercontent.com/8886922/147345045-f2c5e286-4a6d-4b2c-b878-0c5715df2a04.png)

Make sure you double click on the device to see if there is an other one (the roaster) and try again.

Regards

Nico

**Reply by jwarthman** [reply]
Hi Marko and Nico,
Thanks very much for your quick replies!

To Marko's question, yes I had tried the other option briefly. I'm not sure why it didn't appear to work, but today that is the correct driver choice.

And Nico, yes that's one of the pages I've been reading & following. It's very informative, but for me at least, a little unclear how much of the steps are necessary, because:

     UPDATE (3.4.2018): Since v1.2 Artisan can be configured for the Hottop KN-8828B-2K+ with one click. 

In any case, as you suggested the System Information report was VERY helpful. I was able to see not only that the roaster was connected, but also the name of the driver. I then configured Artisan again, and ran a no-beans roast on the Hottop, and Artisan displayed the progress of the roast!

Again, thanks very much for your assistance!

V/R,

Jim
